src/dex/lite-psm/lite-psm-event-pool.ts

```diff
@@ -0,0 +1,216 @@
+import { Interface } from '@ethersproject/abi';
+import { DeepReadonly } from 'ts-essentials';
+import { Contract } from 'web3-eth-contract';
+import { Address, Log, Logger } from '../../types';
+import { StatefulEventSubscriber } from '../../stateful-event-subscriber';
+import { getBigIntPow } from '../../utils';
+import { IDexHelper } from '../../dex-helper/idex-helper';
+import { PoolState, PoolConfig } from './types';
+import { LitePsmConfig } from './config';
+import PsmABI from '../../abi/maker-psm/psm.json';
+import VatABI from '../../abi/maker-psm/vat.json';
+import { erc20Iface } from '../../lib/tokens/utils';
+import DaiABI from '../../abi/lite-psm/dai.json';
+import UsdcABI from '../../abi/lite-psm/usdc.json';
+
+const vatInterface = new Interface(VatABI);
+const psmInterface = new Interface(PsmABI);
+const daiInterface = new Interface(DaiABI);
```

Comment:
@JackieJoo Not clear why we need DAI abi here. Could we use standard erc20 abi?

Reply:
dai has additional events, which we can't parse with erc20 decoder. Same goes for usdc